srcdeps-maven-quickstarts/srcdeps-mvn-git-revision-non-master-quickstart/.mvn/extensions.xml

```diff
@@ -23,4 +23,9 @@
     <artifactId>srcdeps-maven-local-repository</artifactId>
     <version>2.0.2</version><!-- @srcdeps.version@ -->
   </extension>
+  <extension>
+    <groupId>org.srcdeps.mvn</groupId>
+    <artifactId>srcdeps-maven-enforcer</artifactId>
+    <version>2.0.2</version><!-- @srcdeps.version@ -->
```

Comment:
`<!-- @srcdeps.version@ -->` the marker is also used when releasing https://github.com/srcdeps/srcdeps-maven/blob/master/pom.xml#L382 and https://github.com/srcdeps/srcdeps-maven/blob/master/pom.xml#L781